test/runtests.jl

```diff
@@ -20,6 +20,8 @@ include("fdspaces.jl")
 include("fielddiffeq.jl")
 include("hybrid2d.jl")
 include("hybrid3d.jl")
+#include("diffusion2d.jl")
```

Comment:
Not related to this PR, but I think the `diffusion2d.jl` test can be added to the suite to run

Reply:
Oh, I see. This was the second, redundant include of that test. It's already been fixed on `main`, so if you rebase onto main, this will be fixed

Reply:
@jb-mackay this comment is still here, and it would introduce it back to `main`, which we don't want. Probably something missed by the rebase

Reply:
Good eye, will remove